bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml

```diff
@@ -0,0 +1,527 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--Based on hbase-shaded-client 1.7.2, with more transitive exclusions and
+updated transitive versions -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>bigtable-hbase-1.x-parent</artifactId>
+        <groupId>com.google.cloud.bigtable</groupId>
+        <version>2.7.3-SNAPSHOT</version> <!-- {x-version-update:bigtable-client-parent:current} -->
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>hbase-client-1x-compat</artifactId>
+    <description>
+        A repackaged version hbase-shaded-client 1.7.3.
+        This artifact upgrades the shaded contents of hbase-shaded-client to avoid security
+        vulnerabilities found in the Apache version that is now EOL. This is meant to be an
+        alternative for bigtable-hbase endusers that want to continue to use HBase 1 apis with
+        Cloud Bigtable.
+    </description>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+
+        <!-- replace EOL hadoop 2.8.5 with 2.10.2 -->
+        <hadoop.version>2.10.2</hadoop.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-common</artifactId>
+                <version>${hadoop.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-auth</artifactId>
+                <version>${hadoop.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>commons-codec</groupId>
+                <artifactId>commons-codec</artifactId>
+                <version>1.15</version>
+            </dependency>
+            <!-- resolve conflict between hbase-common and hadoop-common -->
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>2.11.0</version>
```

Comment:
please add to renovate if you do not want to bump this version

Reply:
All of the versions here are fair game to update